### ISSUE TITLE
How i can create voice channel

### ISSUE THREAD

**Issue by techuserDMT** [lynoapp/mattermost-plugin-lyno#8]
Hello. I installed lyno plugin in v 1.1.0 for mm v 5.37 and my question is, How i can create a voice channel. 
After install i see something like that. 
![image](https://user-images.githubusercontent.com/88150480/128986085-a3e7f6bc-0b64-4fa5-959e-dc6086fbb754.png)
I don't see panel where i can config it. Please help mi with that.

**Comment by techuserDMT** [comment]
I have deployed mm app on kubernetes cluster in Internal Network. The service is not exposed outside, only in internal network. So i can be the problem of this ?

**Comment by sapkra** [comment]
Exactly, our servers need to communicate with your server to verify your authority. For this we need our plugin endpoint publicly available, that means you need to make it available through a domain.

**Comment by techuserDMT** [comment]
It works. Thank You ;)

**Comment by tonytonyistony** [comment]
Im also having an issue with this. 

 ✓ Site URL
 ✓ Mattermost has to be reachable from the internet - port 8065 only
 ✓ Enable developer set to false.
 ✓ Plugin installed and enabled

The "SOLYD VOICE ROOMS" section has appeared at the top, but no voice channel have appeared. 
How do I tie my MM to my SOLYD account?

**Comment by sapkra** [comment]
@tonytonyistony Can you try out the latest release?
